app/mattermost.js

```diff
@@ -496,7 +497,7 @@ export default class Mattermost {
         }
     };
 
-    onPushNotificationReply = (data, text, badge, completed) => {
+    onPushNotificationReply = (data, text, badge, completed = emptyFunction) => {
```

Comment:
We don't actually need the default value here. `new Promise(func).then(undefined);` doesn't throw an error